packages/core/src/interpreter.ts

```diff
@@ -534,6 +534,34 @@ export class Interpreter<
  * @param machine The machine to interpret
  * @param options Interpreter options
  */
+export function createActor<TMachine extends AnyStateMachine>(
+  machine: AreAllImplementationsAssumedToBeProvided<
+    TMachine['__TResolvedTypesMeta']
+  > extends true
+    ? TMachine
+    : MissingImplementationsError<TMachine['__TResolvedTypesMeta']>,
+  options?: InterpreterOptions<TMachine>
```

Comment:
we probably should also rename all other types that mention the interpreter, for example: `InterpreterOptions` -> `ActorOptions`